src/app-layer-parser.c

```diff
@@ -215,10 +215,16 @@ void AppLayerParserStateFree(AppLayerParserState *pstate)
     SCReturn;
 }
 
+bool g_config_http1keywords_http2traffic = false;
+
 int AppLayerParserSetup(void)
 {
     SCEnter();
     memset(&alp_ctx, 0, sizeof(alp_ctx));
+    int value = 0;
+    if (ConfGetBool("app-layer.protocols.http.rules-on-http2", &value) == 1 && value == 1) {
```

Comment:
I think it would make more sense to make this part of `app-layer.protocols.http2`.

Reply:
indeed, will do